Adjust rendering of widgets: ensure correct rendering

simialbi opened the pull request:
| Q             | A
| ------------- | ---
| Is bugfix?    | no
| New feature?  | no
| Breaks BC?    | no
| Tests pass?   | yes
| Fixed issues  | #1, #70 

I think it's necessary to check all of the form parts at least once again (Do all the layouts work properly and all the fields in all the layouts according to bs4 doc). I'll do that later and I'll define some more tests to check everything of this (WIP).

Additionaly I added a german translation of the guide.

simialbi commented:
I have added the form and field tests in the meantime as mentioned before. Have a look.
I my opinion it now fixes #1 completely. Have a look

samdark commented on docs/guide-de-CH/README.md:
```diff
@@ -0,0 +1,29 @@
+Twitter Bootstrap Erweiterung für Yii 2
```
I think it should be `guide-de`. German in Switzerland should be OK for other German-speaking countries, right? If so, it's better to join efforts.

simialbi replied:
Of course it's nearby the same. There is just one (little) difference the germans have to the swiss (and that's why I named it de-CH): [the ß char](https://en.wikipedia.org/wiki/%C3%9F)
But that's no reason they wont understand 😄 . I'll refactor it.

samdark commented on docs/guide-de-CH/basic-usage.md:
```diff
@@ -0,0 +1,17 @@
+Basic Usage
```
Translation missing.

samdark commented on docs/guide-es/usage-widgets.md:
```diff
@@ -5,19 +5,23 @@ La mayoría de los componentes de bootstrap están encapsulados dentro de Yii wi
+- [[yii\bootstrap4\ButtonToolbar|ButtonToolbar]]
 - [[yii\bootstrap4\Carousel|Carousel]]
-- [[yii\bootstrap4\Collapse|Collapse]]
+- [[yii\bootstrap4\Collapse|Collapse]] *deprecated* (see [[yii\bootstrap4\Accordion|Accordion]])
```
```suggestion
- [[yii\bootstrap4\Collapse|Collapse]] *obsoleto* (see [[yii\bootstrap4\Accordion|Accordion]])
```

samdark commented on docs/guide-ja/usage-widgets.md:
```diff
@@ -4,19 +4,23 @@ Yii ウィジェット
+- [[yii\bootstrap4\ButtonToolbar|ButtonToolbar]]
 - [[yii\bootstrap4\Carousel|Carousel]]
-- [[yii\bootstrap4\Collapse|Collapse]]
+- [[yii\bootstrap4\Collapse|Collapse]] *deprecated* (see [[yii\bootstrap4\Accordion|Accordion]])
```
```suggestion
- [[yii\bootstrap4\Collapse|Collapse]] *廃止予定* (see [[yii\bootstrap4\Accordion|Accordion]])
```

samdark commented on docs/guide-pt-BR/usage-widgets.md:
```diff
@@ -4,16 +4,20 @@ Yii widgets
+- [[yii\bootstrap4\ButtonToolbar|ButtonToolbar]]
 - [[yii\bootstrap4\Carousel|Carousel]]
-- [[yii\bootstrap4\Collapse|Collapse]]
+- [[yii\bootstrap4\Collapse|Collapse]] *deprecated* (see [[yii\bootstrap4\Accordion|Accordion]])
```
```suggestion
- [[yii\bootstrap4\Collapse|Collapse]] *descontinuada* (see [[yii\bootstrap4\Accordion|Accordion]])
```

samdark commented on docs/guide-uk/usage-widgets.md:
```diff
@@ -4,16 +4,20 @@
+- [[yii\bootstrap4\ButtonToolbar|ButtonToolbar]]
 - [[yii\bootstrap4\Carousel|Carousel]]
-- [[yii\bootstrap4\Collapse|Collapse]]
+- [[yii\bootstrap4\Collapse|Collapse]] *deprecated* (see [[yii\bootstrap4\Accordion|Accordion]])
```
```suggestion
- [[yii\bootstrap4\Collapse|Collapse]] *засуджується* (see [[yii\bootstrap4\Accordion|Accordion]])
```

samdark replied:
@SilverFire, @hiqsol, correct?

samdark commented on docs/guide-zh-CN/usage-widgets.md:
```diff
@@ -3,20 +3,23 @@ Yii 小部件
+- [[yii\bootstrap4\ButtonToolbar|ButtonToolbar]]
 - [[yii\bootstrap4\Carousel|Carousel]]
-- [[yii\bootstrap4\Collapse|Collapse]]
+- [[yii\bootstrap4\Collapse|Collapse]] *deprecated* (see [[yii\bootstrap4\Accordion|Accordion]])
```
```suggestion
- [[yii\bootstrap4\Collapse|Collapse]] *弃用* (see [[yii\bootstrap4\Accordion|Accordion]])
```

samdark replied:
@kids-return correct?

simialbi commented:
I changed the translations of `deprecated` like you suggested and copied the Breadcrumbs-Widget from #70  and added a rendering test. I hope we can solve both pr's with these changes.

samdark commented:
Merged. Thank you very much!
